### ISSUE TITLE
SQL Error after install

### ISSUE THREAD

**Issue by edolyne** [solspace/craft3-freeform#2]
After installing freeform lite, the following SQL error was encountered, and the plugin had to be manually disabled via the DB to regain access to the control panel.

```
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'av04121_secondChance.craft_freeform_unfinalized_files' doesn't exist The SQL being executed was: SELECT assetId FROM craft_freeform_unfinalized_files WHERE dateCreated < '2018-01-24 12:18:59'
```

**Comment by gustavs-gutmanis** [comment]
@edolyne could you please check your database and see if *any* freeform tables have been installed at all?

Also, did you install freeform via `Plugin Store` or `composer`?

**Comment by kjmartens** [comment]
This should be resolved now in Freeform 2.0.0-beta.2 :)
Please updates and let me know if the issue persists.
